Frontend/src/app/views/dashboard/project-details/project-details.component.scss

```diff
@@ -1,7 +1,7 @@
+@import "../../../../app/styles/display.scss";
+
 .container {
-    padding-left: 10%;
-    padding-right: 10%;
-    padding-top: 30px;
+    padding: 20px;
 }
```

Comment:
Can probably just remove this and use the padding that the shared `.container` gives